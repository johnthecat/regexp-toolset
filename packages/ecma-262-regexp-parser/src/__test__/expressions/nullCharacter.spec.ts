import { expect, describe, it } from 'vitest';
import { parseRegexp } from '../../index.js';

describe('Null Character', () => {
  it('should parse null character', () => {
    expect(parseRegexp(/\0/)).toMatchSnapshot();
  });
});
