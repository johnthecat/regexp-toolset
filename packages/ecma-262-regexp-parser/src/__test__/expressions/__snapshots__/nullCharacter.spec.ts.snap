// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Null Character > should parse null character 1`] = `
{
  "body": {
    "end": 2,
    "kind": 9,
    "start": 1,
  },
  "end": 3,
  "flags": "",
  "kind": 0,
  "start": 0,
}
`;
